aspnetcore/blazor/host-and-deploy/configure-trimmer.md

```diff
@@ -123,7 +131,7 @@ The component is modified to use the `StringTuple` type:
 + items = JsonSerializer.Deserialize<List<StringTuple>>(data, options)!;
 ```
 
-Because custom types aren't trimmed by Blazor when an app is published (unless explicitly opted in), the component works as designed after the app is published.
+Because custom types aren't trimmed by Blazor when an app is published (and unless [explicitly opted in](#configuration) for class libraries), the component works as designed after the app is published.
```

Comment:
```suggestion
Because custom types defined in non-trimmable assemblies aren't trimmed by Blazor when an app is published, the component works as designed after the app is published.
```